src/Networking/v2/Api.php

```diff
@@ -350,4 +350,354 @@ public function deleteQuota() : array
             ]
         ];
     }
+
+    public function getLoadbalancers(): array
```

Comment:
I thought that look better as well, but changed it to match the API docs. In most of the problems I'm working on, you only use capitalization after an `_`. I'll go through and correct this. Thanks.